src/dishka/container.py

```diff
@@ -1,7 +1,10 @@
+from __future__ import annotations
```

Comment:
why? we have forward ref only in one place

Reply:
`Container.__call__` return `ContextWrapper`, but `ContextWrapper` is created later than `Container`